internal/auth/service_account.go

```diff
@@ -17,7 +18,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const sccName = "log-collector-scc"
+const sccName = "logging-scc"
```

Comment:
It might be not worth the trouble renaming this SCC, unless we know the upgrade scenario deletes the old one and creates the new.

Reply:
I mentioned to @Clee2691 we should create a dedicated SCC for LFME and give it a naming convention that some how maps to the CR name.  This gives us the freedom to deploy multiples in the future though I believe that to be unlikely

Reply:
Since they both use the same SCC I kept the naming but added code to remove the legacy SCC.